pkgs/build-support/node/build-npm-package/hooks/npm-build-hook.sh

```diff
@@ -0,0 +1,26 @@
+# shellcheck shell=bash
+
+npmBuildHook() {
+    echo "Executing npmBuildHook"
+
+    if [ -z "${npmBuildScript-}" ]; then
```

Comment:
We should probably run `preBuild` before checking this, so that can potentially set `npmBuildScript`.

Reply:
@winterqt you marked this as resolved but it doesn't look like it's fixed or addressed.